back_end/personal_project/settings.py

```diff
@@ -28,8 +28,11 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```

Comment:
Is this change for deployment purposes?

Reply:
I just changed it back to []. I don't actually remember changing this, but it may have been when I was troubleshooting my api call.